jobs/awslogs/templates/config/config/README

```diff
@@ -0,0 +1 @@
+# a place for additional config files as specified in awslogsd.sh
```

Comment:
Either way -- I just wanted this dir to exist for any future static configs we might add.